docs/README.md

```diff
@@ -1,58 +1,44 @@
-# VTEX React App Template
+# VTEX Product List Context app
 
-Our guide repository to structure for react apps, that should be used as a template.
+This app is the Product List Context that should be used to store certain products. Example of usage: This is used on the `Shelf` app to store the visible products.
 
-We use `yarn` as our default package manager, before coding make sure to run yarn on: `root` and `react` folders.
+## API
```

Comment:
I wouldn't use ##  API. What do you think about using ## Configuration instead? 

Then, below ## Configuration, I would add the instructions for implementation. How to install it, etc. You can use the Breadcrumb documentation as example (https://github.com/vtex-apps/breadcrumb/tree/master/docs). 

After the instructions, you can add the implementation example used. But remember to clarify this code structure,  what is each value and how it works.

```
import {
  ProductListProvider,
  useProductListState,
  useProductListDispatch,
} from 'vtex.product-list-context/ProductListContext'
...
const { products } = useProductListState()
const dispatch = useProductListDispatch()
return (
  <ProductListProvider>
    <MyComponent />
  </ ProductListProvider>
)
```